packages/block-library/src/cover/index.php

```diff
@@ -47,18 +47,14 @@ function render_block_core_cover( $attributes, $content ) {
 		}
 		$current_featured_image = get_the_post_thumbnail_url();
 
-		$styles = 'background-image:url(' . esc_url( $current_featured_image ) . '); ';
-
-		if ( isset( $attributes['minHeight'] ) ) {
-			$height_unit = empty( $attributes['minHeightUnit'] ) ? 'px' : $attributes['minHeightUnit'];
-			$height      = " min-height:{$attributes['minHeight']}{$height_unit}";
-
-			$styles .= $height;
-		}
```

Comment:
The `min-height` is already serialized in the content. So there's no need to add it separately now that the `style` attribute values are merged.

P.S. We might implement this via the `dimensions` support flag. See #45300.